src/app/js/formats/vega/models/RadarChart.js

```diff
@@ -1,15 +1,19 @@
 import { MONOCHROMATIC_DEFAULT_COLORSET } from '../../colorUtils';
 import { SCALE_LINEAR } from '../../chartsUtils';
+import radarChartVG from './json/radar_chart.vg.json';
+import deepClone from 'lodash.clonedeep';
+import BasicChartVG from './BasicChartVG';
 
 /**
  * Class use for create radar chart spec
```

Comment:
Oh!

Reply:
?

Reply:
The next line says `class RadarChart`, which contains the same information.

Is this comment really informative?